### ISSUE TITLE
encloses method does not always work as expected

### ISSUE THREAD

**Issue by onursumer** [Rescla/range-ts#9]
```JavaScript
const r1 = new NumberRange(3, BoundType.OPEN, 6, BoundType.CLOSED);
const r2 = NumberRange.closed(3, 6);

const thisShouldBeFalse = r1.encloses(r2); // evaluates true but should false
```

`(3..6]` does not enclose `[3..6]`. See  https://guava.dev/releases/19.0/api/docs/com/google/common/collect/Range.html#encloses(com.google.common.collect.Range)

**Comment by Rescla** [comment]
Fixed in 0.1.5, let me know if it works for you.

**Comment by onursumer** [comment]
That was quick, thanks! It works as expected now.
